docs/firebase.md

```diff
@@ -230,3 +230,7 @@ DAQ -> Firebase: Merged (active)
 Firebase -> WebApp: Merged (active)
 note right of WebApp: "Normal" state
 ```
+
+# Firebase Database Schema
+![Diagram](Firebase%20Database.png)
+Generated with [dbdiagram.io](https://dbdiagram.io/) with this [file](firebase_schema)
```

Comment:
can you add an extension to the firebase_schema?  I don't care that it's the right one, necessarily -- just don't want it to be blank, wrong, or misleading.